src/urbit/app/chess.hoon

```diff
@@ -653,9 +654,15 @@
         ?~  archived-game  ~
         ``[%chess-game !>(u.archived-game)]
       ``[%chess-game !>(game.u.active-game)]
-    ::
-    ::  .^(arch %gy /=chess=/game)
-    ::  collect all the game-id keys
+  ::
+  ::  .^(noun %gx /=chess=/mutuals/noun)
+  ::  .^(json %gx /=chess=/mutuals/json)
+  ::  read mutual pals
+    [%x %mutuals ~]
```

Comment:
Though we're targeting the `%mutuals` scry path, we should still be calling it `%pals` or `%friends` on our end